### PR TITLE
headertoolbar component added

### PR DESCRIPTION
**Pull request by Jashpatel1**

### CODE REVIEW

File: frontend/src/components/HeaderToolbar.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template lang="pug">
+    v-app
```
Comment (ajatprabha) [inline frontend/src/components/HeaderToolbar.vue]: There can only be one `v-app` throughout the application.

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,48 +1,17 @@
...
-          v-divider(v-else-if="item.divider" :key="i" dark class="my-4")
-          v-list-item v-else :key="i" @click=""
-            v-list-item-action
-              v-icon({{ item.icon }})
-            v-list-item-content
-              v-list-item-title(class="grey--text") {{ item.text }}
-    v-content
-      router-view
+    v-app
+        v-content
```
Comment (ajatprabha) [inline frontend/src/App.vue:5]: `HeaderToolbar` abstraction looks good but i think it should be kept in `App.vue` only and not included in every view it is required in. Instead you can render in conditionally with `v-if`

File: frontend/src/views/Profile.vue [hunk 3]
```diff
@@ -1,13 +1,16 @@
 <template lang="pug">
     v-row(justify="center")
+        HeaderToolbar
```
Comment (ajatprabha) [inline frontend/src/views/Profile.vue]: This should not be here but jn `App.vue`

**Review by ajatprabha — CHANGES_REQUESTED** [review]

File: frontend/src/components/HeaderToolbar.vue (hunk 1 above)
Comment (Jashpatel1) [reply frontend/src/components/HeaderToolbar.vue]: resolved

File: frontend/src/App.vue (hunk 2 above)
Comment (Jashpatel1) [reply frontend/src/App.vue:5]: done

File: frontend/src/views/Profile.vue (hunk 3 above)
Comment (Jashpatel1) [reply frontend/src/views/Profile.vue]: changed

File: frontend/src/App.vue (hunk 2 above)
Comment (Jashpatel1) [reply frontend/src/App.vue:5]: done

**Pull request merged**